app/views/treehouses/show.html.erb

```diff
@@ -0,0 +1,59 @@
+<div class="container">
+  <h1><%= @treehouse.name %></h1>
+  <%= link_to "Back to all cocktails", root_path %>
+  <br>
+  <%= cl_image_tag @cocktail.photo, height: 300, width: 400, crop: :fill %>
+
+
+  <div class="row">
+    <div class="col-sm-12 col-md-6">
+      <h3>Recipe</h3>
+      <table class="table table-bordered">
+        <tbody>
+          <thead>
+            <th>Ingredient</th>
+            <th>Description</th>
+            <th>Action</th>
+          </thead>
+          <% @cocktail.doses.each do |dose| %>
```

Comment:
cocktails, ingredients etc. this is all copy pasted, please remove all false statements and start over with the stye